src/nomad_simulations/schema_packages/outputs.py

```diff
@@ -310,3 +327,47 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
                     physical_property=phys_property,
                     logger=logger,
                 )
+
+
+class WorkflowOutputs(Outputs):
+    """
+    This section contains output properties that depend on a single system, but were
+    calculated as part of a workflow. Examples include geometry optimization and molecular dynamics.
+    """
+
+    step = Quantity(
+        type=np.int32,
+        description="""
+        The step number with respect to the workflow.
+        """,
+    )
+
+    # TODO add this in when we link to nomad-simulations-workflow schema
+    # workflow_ref = Quantity(
+    #     type=SimulationWorkflow,
+    #     description="""
+    #     Reference to the `SelfConsistency` section that defines the numerical settings to converge the
+    #     output property.
+    #     """,
+    # )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class TrajectoryOutputs(WorkflowOutputs):
+    """
+    This section contains output properties that depend on a single system, but were
```

Comment:
The description is the same as in `WorkflowOutputs`. In fact, what is the role of the `WorkflowOutputs` section?

Reply:
It's just to distinguish workflow outputs that require a step quantity (e.g., geometry optimization) from specific trajectory workflows where there is a physical time quantity...maybe this is not necessary or desired? If I only have 1, I would just name it WorkflowOutputs for generality.